packages/shared/src/components/SNETForgotPasswordSubmit/index.js

```diff
@@ -107,7 +108,6 @@ const SNETForgotPasswordSubmit = props => {
 };
 
 SNETForgotPasswordSubmit.propTypes = {
-  email:PropTypes.string,
   error:PropTypes.string, 
   updateError:PropTypes.func,
```

Comment:
there should not be update error. Refer to the comments in forgotPassword component.